docs/swarm/admin_guide.md

```diff
@@ -0,0 +1,243 @@
+that you carefully place managers in the datacenter topology. For optimal
+fault-tolerance, distribute manager nodes across a minimum of 3 regions to
+support a region failure or maintenance scenario. If you suffer a failure in any
+region, the swarm still has a quorum of manager nodes available to process
+requests.
+
+| Swarm manager nodes |  Repartition (on 3 regions)  |
+|:-------------------:|:----------------------------:|
+| 3                   |            1-1-1             |
+| 5                   |            2-2-1             |
+| 7                   |            3-2-2             |
+| 9                   |            3-3-3             |
+
+## Run manager-only nodes
+
+By default manager nodes also act as a worker nodes. This means the scheduler
+can assign tasks to a manager node. For small and non-critical clusters
+assigning tasks to managers is relatively risk-free as long as you schedule
+services using **resource constraints** for *cpu* and *memory*.
+
+However, because manager nodes use the Raft consensus algorithm to replicate data
+in a consistent way, they are sensitive to resouce starvation. You should
+isolate managers in your swarm from processes that might block cluster
+operations like cluster heartbeat or leader elections.
+
+To avoid interference with manager node operation, you can drain manager nodes
+to make them unavailable as worker nodes:
+
+```bash
+docker node update --availability drain <NODE-ID>
+```
+
+When you drain a node, the scheduler reassigns any tasks running on the node to
+other available worker nodes in the cluster. It also prevents the scheduler from
+assigning tasks on the node.
+
+## Back up the cluster state
+
+Docker manager nodes store the cluster state and manager logs in the following
+directory:
+
+`/var/lib/docker/swarm/raft`
+
+Back up the raft data directory often so that you can use it in case of disaster
+recovery.
+
+You should never restart a manager node with the data directory from another
+node. For example, by copying the `raft` directory from one node to another. The
+data directory is unique to a node ID and a node can only use a given node ID
+once to join the swarm. (ie. Node ID space should be globally unique)
+
+To cleanly re-join a manager node to a cluster:
+
+1. Run `docker node demote <id-node>` to demote the node to a worker.
+2. Run `docker node rm <id-node>` before adding a node back with a fresh state.
+3. Re-join the node to the cluster using `docker swarm join`.
+
+In case of [disaster recovery](#disaster-recovery), you can take the raft data
+directory of one of the manager nodes to restore to a new swarm cluster.
+
+## Monitor swarm health
+
+You can monitor the health of Manager nodes by querying the docker `nodes` API
+in JSON format through the `/nodes` HTTP endpoint. Refer to the [nodes API documentation](https://github.com/docker/docker/blob/master/docs/reference/api/docker_remote_api_v1.24.md#36-nodes)
```

Comment:
proper link here should be:  `../reference/api/docker_remote_api_v1.24.md#36-nodes`